examplePrograms/library_components_export.py

```diff
@@ -0,0 +1,129 @@
+from nexarClients.design.nexarDesignClient import NexarClient
+import os
+import json
+import csv
+
+# This is the API query, feel free to chop and change this to suit your use case.
+# We recommend building queries in an IDE such as BananaCakePop or Postman.
+# You can use BananaCakePop in a browser here: https://api.nexar.com/graphql/
+
+workspaceURL = "Your workspace URL here"
```

Comment:
The casing of Python variables within this file is a bit inconsistent:
- `workspaceURL`, `manufacturerPart` - camelCase
- `client_id`, `client_secret`, `page_info`, etc. - snake_case

It's a minor thing, but Python variables and function names are usually all snake_case so it'd be good to apply this consistently.